torch/testing/_internal/distributed/rpc_utils.py

```diff
@@ -73,7 +93,11 @@ def subprocess_init(self, data):
         sys.stderr = os.fdopen(2, "wt")
 
     def subprocess_env_vars(self):
-        env_vars = self.get_env_vars()
+        env_vars = dict()
+        if self.LIBSEGFAULT_PATH is not None:
+            env_vars["LD_PRELOAD"] = self.LIBSEGFAULT_PATH
+            env_vars["SEGFAULT_SIGNALS"] = "segv abrt bus"
```

Comment:
Can we do this for all distributed tests instead of just RPC tests by adding this to MultiProcessTestCase instead?